### ISSUE TITLE
Add Logging Module

### ISSUE THREAD

**Issue by InValidFire** [InValidFire/ghau#6]
Currently for errors and update status, ghau will simply print a message to the console. I would like to expand this using the built-in python `logging` module. If I understand this module correctly, it will provide the ability to receive the messages in any way they decide useful for their program.

I currently do not have any experience regarding the logging module, so I would be welcome to assistance regarding implementing this feature.

**Comment by Space-Turtle0** [comment]
So your basically trying to merge the logging module with ghau? Anything else in particular?

**Comment by InValidFire** [comment]
I want to utilize the logging module for emitting messages so they may be collected and stored by the applications using the ghau module.

**Comment by Space-Turtle0** [comment]
This is an example I got, maybe something like this?
```
import logging
import ghau

log = logging.getLogger(__name__) 
log.addHandler(logging.NullHandler()) 
  
# Example function (for testing) 
def func(): 
    log.critical('A Critical Error !') 
    log.debug('A debug message') 

logging.basicConfig(level=logging.LEVEL) 
#Replace LEVEL with a logging level.
ghau.func

#Example
logging.basicConfig(level=logging.DEBUG) 
logging.debug('Logged')
#The str above will get logged. 


#Log to a file:
logging.basicConfig(
  filename='app.log',
  filemode ='a'
  format ='%(name)s - %(levelname)s - %(message)s'
)

logging.warning("This will get logged to a file")
#Output: root - WARNING - This will get logged to a file
```

**Comment by InValidFire** [comment]
Now closed as of commit 9378b48a0fa55d79a128e0e1ce9174c85aa17de0
